executor/show_test.go

```diff
@@ -1001,3 +1002,28 @@ func (s *testSerialSuite1) TestShowCreateTableWithIntegerDisplayLengthWarnings(c
 		"  `e` bigint DEFAULT NULL\n" +
 		") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin"))
 }
+
+func (s *testSuite5) TestShowVar(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	var showSQL string
+	for _, v := range variable.SysVars {
+		if v.Scope != variable.ScopeNone && v.Scope != variable.ScopeGlobal|variable.ScopeSession {
```

Comment:
Please add cases for variables with `ScopeGlobal | ScopeSession` or `ScopeNone`.

Reply:
Done.